### ISSUE TITLE
Feature Request: for differently sized creatures

### ISSUE THREAD

**Issue by Iankid** [erithtotl/FVTT-Wall-Height#4]
hey, was wondering about a possible feature addition.
I got some PC's who are large sized, and thus want to treat as being able to see over 5 ft. walls.
I don't, however, want to keep setting their position to be 5 ft. in the air.
Think there's some way to mark some tokens as being higher then the wall, without affecting their elevation?

**Comment by erithtotl** [comment]
Yeah I've been thinking about this as well.  I could auto calculate this but some large creatures can be rather low to the ground (like a giant centipede).  In order for it to work was considering adding a field on the token for token height...

**Comment by GambetTV** [comment]
I was coming here to suggest the same thing. Maybe a field in Token Configuration would work.
